benchmark/run.js

```diff
@@ -30,11 +40,17 @@ if (benchmarks.length === 0) {
   child.on('message', function(data) {
     // Construct configuration string, " A=a, B=b, ..."
     let conf = '';
+    let confseparator = ' ';
+    let separator = ': ';
+    if (data.addOpts.format == 'csv') {
+      confseparator = ',';
+      separator = ',';
+    }
     for (const key of Object.keys(data.conf)) {
-      conf += ' ' + key + '=' + JSON.stringify(data.conf[key]);
+      conf += confseparator + key + '=' + JSON.stringify(data.conf[key]);
     }
 
-    console.log(`${data.name}${conf}: ${data.rate}`);
+    console.log(`${data.name}${conf}${separator}${data.rate}`);
```

Comment:
Could you add `data.time` to the csv output and surround `data.name` with `"` just to be safe. I would suggest:

```js
if (format === 'csv') {
  // Escape qoutes (") for correct csv formatting
  conf = conf.replace(/"/g, '""');
  console.log(`"${data.name}", "${conf}", ${data.rate}, ${data.time}`);
} else {
  console.log(`${data.name} ${conf}: ${data.rate}`);
}
```